core/tauri/src/lib.rs

```diff
@@ -154,24 +154,48 @@ impl<A: Assets> Context<A> {
     self.assets.clone()
   }
 
+  /// A mutable reference to the assets to be served directly by Tauri.
+  #[inline(always)]
+  pub fn assets_mut(&mut self) -> &mut Arc<A> {
```

Comment:
why a mutable getter for an Arc type?

Reply:
So that they can mutate the actual assets if their assets type doesn't have interior mutability. `Arc` provides [get_mut()](https://doc.rust-lang.org/std/sync/struct.Arc.html#method.get_mut) to turn a mutable arc into a single mutable reference